### DISCUSSION TITLE
Large difference between the energy predictions from CHGNet and VASP

### DISCUSSION THREAD

**Discussion by kaijiz** [CederGroupHub/chgnet#217] [Q&A] [answered]
<img width="739" alt="Screen Shot 2024-12-04 at 10 32 29 PM" src="https://github.com/user-attachments/assets/0e835c58-8af8-40df-a74c-531909889b54">
Hi, i recently want to do some test about the accuracy of CHGNet, so i choose some structures and test the difference of energies predicted by CHGNet and VASP. The result is shown below. You can find it's not very close. CHGNet always obtains a lower energy. I don't know why, maybe i used different settings in VASP? 
Below is the CHGNet code and INCAR  i use. Please give me some advice. Thanks so much!
![Screen Shot 2024-12-04 at 10 39 08 PM](https://github.com/user-attachments/assets/f914909e-f14c-43f9-82a9-cfbb31388df7)
![Screen Shot 2024-12-04 at 10 39 42 PM](https://github.com/user-attachments/assets/c63d1d1c-063f-4425-805a-96f33c0334bd)

**Comment by kaijiz** [comment]
![Screen Shot 2024-12-04 at 10 39 08 PM](https://github.com/user-attachments/assets/cab6975f-32ce-4625-b23e-9131ab79d781)

**Comment by kaijiz** [comment]
![Screen Shot 2024-12-04 at 10 39 42 PM](https://github.com/user-attachments/assets/f036ab4e-5dcf-4766-85e8-19f00af41058)

**Reply by bowen-bd** [reply, marked as answer]
This is due to the MP comparability. CHGNet energies are MPRelaxSet+compatibility energies.
Check

https://matsci.org/t/mp-compatibility-and-energy-corrections/35035

**Reply by kaijiz** [reply]
Thank you so much!
